tools/common/utils.go

```diff
@@ -21,6 +24,22 @@ const (
 	TxStatusOk = "ok"
 )
 
+func GetActorNameFromAddress(helper *helper.Helper, addr address.Address, height int64, tipsetKey filTypes.TipSetKey) (string, error) {
+	// #nosec G115
+	actorName, err := helper.GetActorNameFromAddress(addr, height, tipsetKey)
+	if err != nil {
+		if errors.Is(err, cache.ErrBadAddress) {
+			// the bad address may have been set due to a previous failed request to the node
+			// clear the cache to allow trying again when reprocessed
+			helper.GetActorsCache().ClearBadAddressCache()
+		}
+
+		return "", fmt.Errorf("could not get actor name from address. err: %w", err)
+	}
+
+	return actorName, nil
+}
+
```

Comment:
I have a question here. From what I saw, the current address that it is failing on calibration is not really valid (never seen on the network). Is this going to work anyway?